Drivers/RcReceiver/Src/RCReceiver_config.cpp

```diff
@@ -0,0 +1,7 @@
+#include "RCReceiver_config.hpp"
```

Comment:
I might be brainfarting, was this method of creating an instance defined somewhere? @Chrisc110 Im not sure what was decided about where/how to make instances of drivers.

Reply:
We do have a schema defined now. @antimonyB finalized it yesterday.